lib/msal-common/src/authority/Authority.ts

```diff
@@ -491,6 +500,10 @@ export class Authority {
                     instanceDiscoveryEndpoint,
                     options
                 );
+            const httpVer = response.headers["x-ms-httpver"];
+            if (httpVer) {
+                this.performanceClient?.setHttpVer(httpVer);
```

Comment:
Readers the http version headers and sets the value

Reply:
Could this theoretically change between requests? I don't think we should assume that any property of a given request for a user/session will be static (i.e. this should be set on a per request basis, not per session).

Reply:
Instead of attaching this as a property on the performance client, how about creating a submeasurement for the instance discovery metadata call (and potentially other network calls, e.g. token endpoint calls), and then adding this as a property on the top-level measurement? Especially since I figure we would want this information not only for the instanceDiscovery call, but also token endpoint calls).

Reply:
Great point @jasonnutter , the http version 2 was enabled by Gateway team on login.microsoftonline.com so I used it to test it out and attached the http version for the discovery metadata call . 
Are we intending to have two http version header columns in perf data in this case , one for /token and another for /authorize? If so, it would make sense to include it as part of the submeasurements.

Reply:
We only load the authorize endpoint in an iframe, where I don't think we would be able to inspect response headers, so it would be for the metadata endpoints and token endpoint.